handlers/group.ts

```diff
@@ -11,46 +12,49 @@ interface IGroup {
 
 class GroupHandle implements IGroup {
     readonly gun: Gun
-    constructor(private db: Gun) {
+    constructor(db: Gun) {
         this.gun = db
     }
 
     public create(groupName: string, callback : () => void): void {
         let user = this.gun.user(userPub)
-        let groupId : string = randomUUID();        
-        this.gun.get("groups").get("groupId").set({groupId: groupId})
+        const groupId : string = uuid();
+        this.gun.get("groups").get("groupId").set(groupId)
         let context = this.gun.get("groups").get("groupId").get(groupId)
-        context.get("members").set({members: userPub})
+        context.get("members").set(userPub)
```

Comment:
if you really want to rework this part, why not just set `context.get("members").set(user)`, so we get the actual benefits of a graph database pointing at the actual user space objects, or does this do the same

Reply:
Har samme funktionalitet som før med et objekt  mindre. Hvis det er dårlig databasebrug (quite possibly) er det et problem der skal fikses mange steder.